test/cql-pytest/conftest.py

```diff
@@ -268,8 +268,12 @@ def has_tablets(cql):
 @pytest.fixture(scope="function")
 def skip_without_tablets(scylla_only, has_tablets):
     if not has_tablets:
-        pytest.skip("Test needs tablets experimental feature on")
+        pytest.skip("Test needs tablets feature on")
 
+@pytest.fixture(scope="function")
+def skip_with_tablets(has_tablets):
+    if has_tablets:
+        pytest.skip("Test may crash with tablets feature on")
```

Comment:
It was a commit that was left from the previous implementation of this PR. I thought it may be needed one day, but I also agree that if it's needed, it should be added when it's needed, not now. So I dropped this commit.